source/refills/tooltipped.html.erb

```diff
@@ -0,0 +1,7 @@
+<a href="#0" class="tooltipped" aria-label="View Notifications">
+  <svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
@tysongach Do you not feel that's covered by "Tooltips are used to add descriptive text to interactive elements which are otherwise lacking descriptive text, like icon links and buttons"?